index.html

```diff
@@ -30,26 +30,172 @@ <h1>
+            sign-ups required.
+          </p>
+          <ul class="work-lang">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+        <article class="work workHeight">
+          <img src="assets/Snapshoot Portfolio-2.svg" alt="" />
+          <h3>Multi-Post Stories</h3>
+          <ul class="skill-info">
+            <li>CANOPY</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-lang">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+        <article class="work">
+          <img src="assets/Snapshoot Portfolio-3.svg" alt="" />
+          <h3>Tonic</h3>
+          <ul class="skill-info">
+            <li>CANOPY</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-lang">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+        <article class="work workHeight">
+          <img src="assets/Snapshoot Portfolio-4.svg" alt="" />
+          <h3>Multi-Post Stories</h3>
+          <ul class="skill-info">
+            <li>CANOPY</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-lang">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+      </section>
+
+      <section class="about-section">
+        <article>
+          <h2>About Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div class="contactTitle"><a href="#">LET’S CONNECT</a></div>
```

Comment:
- [ ] KIndly make sure the `LET'S CONNECT` title is rendered on the page using heading tags `h2, h3, h4, hx. ..` in order to follow best HTML practices and meet the project's requirements a well.